docs/index.md

```diff
@@ -8,9 +8,9 @@ description: |-
 
 # Terraform Provider for VMware Cloud Foundation
 
-This provider works with VMware Cloud Foundation 4.4.0 to 4.5.2
+This provider works with VMware Cloud Foundation 4.4.0 to 5.1.1
```

Comment:
Maybe you add that legacy versions up to v0.8.5 work with VCF 4.4.0 to 4.5.2.

Reply:
The docs are versioned on the registry